Frontend/ui-common/src/Components/RateCard/View.purs

```diff
@@ -318,6 +320,45 @@ fareUpdateView push config =
       ]
   ]
 
+tollOrParkingView :: forall w. (Action -> Effect Unit) -> Config -> PrestoDOM (Effect Unit) w
+tollOrParkingView push config = 
+  linearLayout
+  [ width MATCH_PARENT
+  , height WRAP_CONTENT
+  , orientation VERTICAL
+  , padding $ Padding 20 0 20 160
+  ][
+    textView $[ 
```

Comment:
Use common textView here,
These lines can be replaced by 4 lines by using commonTV